packages/realm/src/app-services/EmailPasswordAuth.ts

```diff
@@ -34,7 +34,7 @@ export class EmailPasswordAuth {
   /**
    * Registers a new email identity with the email/password provider,
    * and sends a confirmation email to the provided address.
-   * @param details The new user's email and password details
+   * @param details - The new user's email and password details
```

Comment:
The first missing `.` I've found so far 😉 
```suggestion
   * @param details - The new user's email and password details.
```